<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="STSLP95" ignored="0" name="bugzilla Test" tests="7" failures="1" timestamp="02 Jan 2020 11:37:37 GMT" time="189.837" errors="0">
  <testcase name="loginTest" time="4.919" classname="com.bugzilla.testCases.TC_LoginTest_001"/>
  <testcase name="loginDDT" time="6.503" classname="com.bugzilla.testCases.TC_LoginDDT_002"/>
  <testcase name="loginDDT" time="7.578" classname="com.bugzilla.testCases.TC_LoginDDT_002"/>
  <testcase name="loginDDT" time="7.109" classname="com.bugzilla.testCases.TC_LoginDDT_002"/>
  <testcase name="loginDDT" time="5.799" classname="com.bugzilla.testCases.TC_LoginDDT_002"/>
  <testcase name="loginDDT" time="4.816" classname="com.bugzilla.testCases.TC_LoginDDT_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.bugzilla.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:39)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="addNewUser" time="33.472" classname="com.bugzilla.testCases.TC_addNewUser_Test_004"/>
</testsuite> <!-- bugzilla Test -->
